sdk/java/src/main/java/com/chain/http/Client.java

```diff
@@ -61,8 +62,7 @@ public Client() {
     }
 
     this.url = url;
-    this.httpClient = new OkHttpClient();
```

Comment:
I'm a tad confused why these two lines changed. Just out of curiosity, why were we not using `this.defaultHttpClient()` prior to this change?